### PR TITLE
qemu_disk_img: Add sync in windows

### PR DESCRIPTION
**Pull request by pingl2016**
Support to sync data in windows guest.

Signed-off-by: Ping Li pingl@redhat.com

ID: 1390069

### CODE REVIEW

**Comment by pingl2016** [comment]
@CongLi @xiagao Could you help to review this PR?

File: qemu/tests/qemu_disk_img.py [hunk 1]
```diff
@@ -95,14 +96,29 @@ def __md5sum(self, dst):
         md5 = re.findall("\w{32}", output)[0]
         return md5
 
+    def set_winutils_letter(self, cmd, session, params):
```
Comment (CongLi) [inline qemu/tests/qemu_disk_img.py]: I think function name and return do not match.

**Comment by CongLi** [comment]
Looks good.

Acked-by: Cong Li <coli@redhat.com>

**Comment by xiagao** [comment]
Run with win7 guest(i386 and x86)  and win2016.x86 and the results are pass.
(1/2) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.Win7.i386.sp1.io-github-autotest-qemu.qemu_disk_img.commit.cache_mode.none: PASS (292.71 s)
 (2/2) smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.qemu_disk_img.commit.cache_mode.none: PASS (436.46 s)

1-smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.Win2016.x86_64.io-github-autotest-qemu.qemu_disk_img.commit.cache_mode.none Finshed 2016-11-01 03:02:05 2016-11-01 03:09:06 PASS 421.041224

**Comment by xiagao** [comment]
Acked-by: Xiaoling Gao <xiagao@redhat.com>

**Comment by yanan-fu** [comment]
LGTM
Test PASS with Win10.x86_64. 
(1/1) smp_8.8192m.repeat1.Host_RHEL.m6.u8.spice.qcow2.virtio_scsi.up.virtio_net.Win10.x86_64.io-github-autotest-qemu.qemu_disk_img.commit: PASS (242.79 s)

Acked by: Yanan Fu  yfu@redhat.com

**Comment by PandaWei** [comment]
ACK

**Comment by CongLi** [comment]
Thanks all.

**Pull request merged**